client/webserver/site/src/js/markets.ts

```diff
@@ -802,6 +816,9 @@ export default class MarketsPage extends BasePage {
       Doc.bind(bttn, 'click', () => this.candleDurationSelected(dur))
       page.durBttnBox.appendChild(bttn)
     }
+
+    // load candlesticks here since we are resetting page.durBttnBox above.
+    this.loadCandles()
```

Comment:
Oh yeah, `page.durBttnBox` needs to be populated in order for `loadCandles` to apply the `selected` class.  Seems fine but have not re-tested.